lib/middleware.js

```diff
@@ -131,6 +127,27 @@ module.exports = function wrapper(context) {
 
         resolve();
       }
+
+      if (
+        context.options.lazy &&
+        (!context.options.filename || context.options.filename.test(filename))
+      ) {
+        context.rebuild();
+      }
+
+      if (HASH_REGEXP.test(filename)) {
+        try {
+          if (context.fs.statSync(filename).isFile()) {
+            processRequest();
+
+            return;
+          }
+        } catch (_error) {
+          // Ignore error
```

Comment:
Is this ok if we ignore this error?

Reply:
Yes, we should check `html` index file if we can't find bundle, so we need ignore this error, maybe we can change logic in future version